spec/cppclass/tensor.yaml

```diff
@@ -229,8 +229,8 @@ methods:
 - smm(Tensor  mat2) -> Tensor
 - softmax(int64_t dim, ScalarType dtype) -> Tensor
 - softmax(int64_t dim) -> Tensor
-#- split(int64_t split_size, int64_t dim=0) -> std::vector<Tensor>
-#- split_with_sizes(IntArrayRef split_sizes, int64_t dim=0) -> std::vector<Tensor>
+- split(int64_t split_size, int64_t dim=0) -> TensorList
+- split_with_sizes(IntArrayRef split_sizes, int64_t dim=0) -> TensorList
```

Comment:
This codes is not related to backward-operations.
I will drop this codes for the PR.

Reply:
It may not be a problem. I leave it as it is.